### ISSUE TITLE
[BUG] controller.flipCamera() work slow on iOS

### ISSUE THREAD

**Issue by c0c4i** [juliuscanute/qr_code_scanner#182]
I had make a fast app for test with home screen with button that when you press it change page to new page with `QRView`.
I need to use QR Scanner with the front camera so I have to flip the camera

As you said in example, I'm doing this to flip the camera
```dart
void _onQRViewCreated(QRViewController controller) {
    this.controller = controller;
    controller.flipCamera();
    controller.scannedDataStream.listen((scanData) {
      print(scanData);
    });
  }
```
On Android it works perfectly opening instantly camera and scanning for QR Code, on the other side on iOS when I press the button it stay on home screen for 3/4 seconds and after it change page and go to QRView page.

In my test it's slow on iOS just when it has inside controller `controller.flipCamera()`, without this it works perfectly.

The workaround to keep it fast is to use
```dart
Future.delayed(Duration(seconds: 1), () {
      controller.flipCamera();
    });
```

**Comment by juliansteenbakker** [comment]
Thank you for reporting this. I'll look into it.

**Comment by naseeb009** [comment]
Hello, Any update regarding this?

**Comment by juliansteenbakker** [comment]
I have released V0.3.0 which takes a different approach to flipping the camera. Can you please check if this issue still persists?

**Comment by c0c4i** [comment]
I test v0.3.0 and it's now perfect on iOS, it's open instantly thanks 👍🏻
